Dockerfile

```diff
@@ -63,6 +63,19 @@ RUN go get github.com/svent/sift@$SIFT_VERSION \
 RUN go get github.com/securego/gosec/cmd/gosec@915e9ee \
   && mv /root/go/bin/gosec /usr/bin/
 
+### sgrep tool install https://sgrep.dev
+ENV SGREP_VERSION 0.4.9b5
```

Comment:
I'm going to cut a stable release but wanted to get feedback on the overall PR first